### PR TITLE
[WIP][RPM] Update foreman-tasks to 0.9.1 and foreman-tasks-core to 0.1.2

### PR DESCRIPTION
**Pull request by adamruzicka**
Requires https://github.com/theforeman/foreman-packaging/pull/1583

For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.14
* [ ] 1.13
* [ ] 1.12

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: rubygem-foreman-tasks/rubygem-foreman-tasks.spec [hunk 1]
```diff
@@ -50,7 +50,7 @@ BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks-core)
-BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 0.8.17
+BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 0.8.22
```
Comment (domcleal) [inline rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: This dynflow build dependency is provided within this PR, please submit a PR with it separately, first.

File: rubygem-foreman-tasks/rubygem-foreman-tasks.spec [hunk 2]
```diff
@@ -26,7 +26,7 @@ Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
 Requires: foreman >= 1.9.0
```
Comment (domcleal) [inline rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: Changed to 1.15.0, ditto for the BuildRequires.

**Comment by domcleal** [comment]
I'm unsure why this has been updated without any reply to my comments, setting it back to WoC until there's a response.

**Comment by adamruzicka** [comment]
I updated the foreman dependency. For the dependent plugins I need to reach out to their maintainers.  Please leave it as WoC until then.

**Comment by iNecas** [comment]
[x] - foreman_salt https://github.com/theforeman/foreman-packaging/pull/1593
[x] - remote_execution https://github.com/theforeman/foreman-packaging/pull/1588 - needs this PR first, therefore it should not block this PR
[ ] - foreman_ansible https://github.com/theforeman/foreman-packaging/pull/1597

Since all the plugins have already a PR for packaging, I'm suggesting proceeding with review/merge, especially when we have chicken&egg problem

**Comment by mmoll** [comment]
As long as #1597 isn't merged, merging/building this PR would lead the repoclosure jobs to fail, so #1597 is mandatory to get in first.

**Comment by domcleal** [comment]
This PR may be updated now.

**Comment by iNecas** [comment]
Anything left unaddressed or can we merge?

**Comment by adamruzicka** [comment]
As far as I remember I addressed all the comments I could fix here before this got blocked by dependent plugins

File: rubygem-foreman-tasks/rubygem-foreman-tasks.spec (hunk 1 above)
Comment (iNecas) [inline rubygem-foreman-tasks/rubygem-foreman-tasks.spec]: Done is https://github.com/theforeman/foreman-packaging/pull/1583

**Comment by domcleal** [comment]
The PR stated "WIP" still, so it was unclear. Merged, thanks @adamruzicka.

**Comment by adamruzicka** [comment]
@domcleal Sorry about that, thank you for bearing with me

**Pull request merged**